rasa_core/trackers.py

```diff
@@ -79,10 +79,16 @@ def current_state(self, should_include_events=False):
         else:
             events = None
 
+        latest_message_time = None
+        timestamps = [e.time for e in self.events if hasattr(e,'time')]
+        if len(timestamps) > 0:
+            latest_message_time = max(timestamps)
```

Comment:
is there any case where the events are not sorted by timestamps?

Reply:
ah you mean take the last of the events rather than the max? no I suppose there's no reason they wouldn't be sorted

Reply:
cool - if we add the time to all events the last one should have the timestamp :+1: